<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDatabaseDialogClass</class>
 <widget class="QDialog" name="CreateDatabaseDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateDatabaseDialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="bufferSizeInMBLabel">
      <property name="text">
       <string>BufferSize(in MB)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="bufferSizeInMBLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="pageSizeLabel">
      <property name="text">
       <string>PageSize</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pageSizeLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="databaseNameLabel">
      <property name="text">
       <string>Database Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="databaseNameLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="numberOfPagesLabel">
      <property name="text">
       <string>Number of Pages</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="numberOfPagesLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
